src/templates/ChangelogList.js

```diff
@@ -4,6 +4,17 @@ import { SEO, TextLink as Link, SitewideHeader, SitewideFooter } from 'component
 import { SimpleCenteredHeading } from 'components/landing';
 import { ChangeSet, Pagination } from 'components/changelog';
 
+// In the code below, this is used to prevent the changelog from being collapsed on the preview
+// site. There is basic authentication on the preview site (preview.roadie.io) to prevent it being
+// indexed by search engines. This basic authentication is preventing prefetching from working,
+// because the requests to prefetch content do not have the authentication information added.
+// For some reason I (David) cannot understand, the ChangeSet componentis being chunked and
+// prefetched. On the preview site, it never gets loaded because of the prefetch authentication
+// problem, and I cannot expand the changelog items to check that the content looks as expected.
+// For this reason, I am using an env var to toggle off collapsing on the preview site but
+// nowhere else.
+import { isPreviewSite } from '../environment';
```

Comment:
The hack is explained in the comment. On preview.roadie.io, the following resources 401:

![Screenshot 2022-03-14 at 23 44 19](https://user-images.githubusercontent.com/562403/158278678-c3f80c48-52b2-4ee9-a450-a152403eb81e.png)

This correlates with the prefetch items in the head.

![Screenshot 2022-03-14 at 23 44 47](https://user-images.githubusercontent.com/562403/158278727-57471f6a-6644-45a9-844e-b52ee1930dad.png)

I read many docs but I couldn't come up with a way to make these resources be included in the initial bundle.

Reply:
Are we able to turn off prefetching globally for the preview site with this: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#disableCorePrefetching?

Reply:
I don't fully understand the docs but that seems to only be available inside Gatsby plugins. I've seen a few GitHub issues where people are asking how to turn off prefetching and Gatsby support basically say that you can't.